### ISSUE TITLE
How "Real Time" should RTSP be?

### ISSUE THREAD

**Issue by jschwalbe** [cdgriffith/pi_streaming_setup#4]
Greetings!! Thanks for the cool stuff you've done with this project.

I've got a Pi3b with a USB webcam plugged in, connected via WiFi. It is streaming correctly to the /streaming/ url. But it's about 19 seconds behind. Is that normal or should I be looking for a problem? Thanks!

**Comment by jschwalbe** [comment]
Actually, I guess this isn't RTSP. Please see the error below, seems as though RTSP isn't working:

```
pi@raspberrypi:~ $ sudo python3 streaming_setup.py --rtsp
Starting streaming_setup 1.5.0
Using arguments: {'version': False, 'ffmpeg_command': False, 'device': '/dev/video0', 'video_size': '1920x1080', 'rtsp': True, 'rtsp_url': '', 'input_format': 'h264', 'codec': 'copy', 'ffmpeg_params': None, 'index_file': '/var/lib/streaming/index.html', 'on_reboot_file': '/var/lib/streaming/setup_streaming.sh', 'systemd_file': '/etc/systemd/system/stream_camera.service', 'compile_ffmpeg': False, 'camera_info': False, 'minimal': False, 'run_as': 'root', 'disable_fdk_aac': False, 'disable_avisynth': False, 'disable_dav1d': False, 'disable_zimg': False, 'disable_kvazaar': False, 'disable_libxavs': False, 'disable_libsrt': False, 'rebuild_all': False, 'safe': False}
ffmpeg already installed, skipping
Traceback (most recent call last):
  File "streaming_setup.py", line 877, in <module>
    main()
  File "streaming_setup.py", line 862, in main
    install_rtsp()
  File "streaming_setup.py", line 746, in install_rtsp
    rtsp_assets = json.dumps(urlopen(rtsp_releases[0]["assets_url"]).read().decode('utf-8'))
TypeError: string indices must be integers
```

Any ideas where to start?

**Comment by cdgriffith** [comment]
Hi @jschwalbe so sorry for the delay, I apparently was not "watching" this repo so I never got notifications! 

Thankfully rezrov has already created the PR that fixed the issue and has been fixed in [1.5.1](https://github.com/cdgriffith/pi_streaming_setup/releases/tag/1.5.1)

For how long the RTSP deplay should be, I am getting around 1~3 seconds on my various cameras. I am seeing a much longer delay, 10~20 for DASH and HLS so have been sticking with RTSP myself.
